src/Tabs/Tabs.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import './Tabs.css';
+
+class Tabs extends React.Component {
+  state = {
+    tab: 0,
```

Comment:
давай немного очевиднее
```suggestion
    activeTab: 0,
```
```suggestion
    currentTab: 0,
```